src/ai/mcts.rs

```diff
@@ -364,8 +364,7 @@ pub fn mcts_build_tree<B: AltBoard>(
 
     let mut tree = Tree::new(root_board.clone());
 
-    let root_outcome = root_board.outcome().map(|o| o.pov(root_board.next_player().other()));
-    tree.nodes.push(Node::new(None, root_outcome));
+    tree.nodes.push(Node::new(None, None));
```

Comment:
Why this change? I think this introduces a bug!

Reply:
Oops, this was not supposed to go with it.
I changed it to this because `root_board` is not done, and the outcome will always be None

Reply:
Oh you're right, there's an assert right above that checks that the board is not done.

Ideally the all the MCTS stuff would keep working even if the root board is done, and just return `None` in all the right places, but that's indeed a separate issue.